api/lib/Gateway.ts

```diff
@@ -1336,6 +1336,10 @@ export default class Gateway {
 						valueId.property,
 						valueId.propertyKey,
 					)
+					if (valueId.value?.unit) {
+						cfg.discovery_payload.value_template =
+							"{{ value_json.value.value | default('') }}"
```

Comment:
I think you have a typo here? 

```suggestion
							"{{ value_json.value | default('') }}"
```

Reply:
No, this is intentional, and the critical part of this chain. Example data from MQTT:

`zwave/Garage/Loft_Stairwell_Lights/38/0/duration`
```json
{"time":1726797698474,"value":{"value":0,"unit":"seconds"}}
```

Notice value is nested within value.

Reply:
Oh didn't know about that! Thanks for clarification